src/test/java/uk/gov/companieshouse/extensions/api/reasons/ReasonServiceUnitTest.java

```diff
@@ -192,8 +187,8 @@ public void testReasonIsRemovedFromRequest() {
 
         FileTransferApiClientResponse response = new FileTransferApiClientResponse();
         response.setHttpStatus(HttpStatus.NO_CONTENT);
-        when(fileTransferApiClient.delete("1234")).thenReturn(response);
-        when(fileTransferApiClient.delete("5678")).thenReturn(response);
+        lenient().when(fileTransferApiClient.delete("1234")).thenReturn(response);
```

Comment:
Why are we using lenient here? Can we not just remove the stubbing?
(Same with any other uses of lenient)

Reply:
"Remove the stubbing" - do you mean remove the second line of stubbing?

Reply:
Similarly to what Mark said really, either something has broken, or the stubbing was never needed (I know Junit5 won't let you have unused when statements, not sure if junit4 would allow you to)

Reply:
it's mockito that will complain about unnecessary stubbings. Those whens should still be needed if the code being tested has not changed.

Reply:
Not sure about this then we can either remove lenient or remove the second when.

Reply:
We shouldn't have to remove or add any when's unless we're making changes to the class being tested or we've changed the input data which may cause a different route through the method to be taken. Mockito will throw an error if there is a when which is not used as has happened here. It was being used but something has broken it in this PR so you'll have to debug it to work out what has gone wrong.

Reply:
mockito also throws an error when there are two mock method calls for the same method - the new extend with runner does not like any ambiguity or potential clashes - the opinions on the forms seem to be to use when then anser instead as it alows for multiple possibilities - debugged with the and both parameters turn up in the code as expected.